tests/application/ingress/application-is-internal-vanity.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: skiperator.kartverket.no/v1alpha1
+kind: Application
+metadata:
+  name: internalvanity
+spec:
+  image: image
+  port: 8080
+  ingresses:
```

Comment:
Would be nice to have a couple more tests:
- internal flag missing
- error handling